index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Tribute Page Built by Sheldon and Muda</title>
+    <link rel="stylesheet" href="index.css">
+</head>
+<body>
+    <main id="main">
+        <h1 id="title">Nelson Mandela</h1>
```

Comment:
I think it would be better to align the heading at the center,to make your page look better.